LongitudinalDifferenceInDifferencesPy/didDesigner.py

```diff
@@ -261,27 +265,30 @@ def printTreatmentMatrixPoints(self, df):
         treatmentMatrix = self._calculate_aggregate_treatment_matrix(df, aggregate=np.mean)
         kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = \
             treatmentMatrix
-        print('pre-treatment mean KPI of comparison group: ' + kpiTreat0Post0_mean + \
-            '\npost-treatment mean KPI of comparison group: ' + kpiTreat0Post1_mean + \
-            '\npre-treatment mean KPI of treated group: ' + kpiTreat1Post0_mean + \
-            '\npost-treatment mean KPI of treated group: ' + kpikpiTreat1Post1_mean)
+
+        print('{ Pre-treatment mean KPI of comparison group: ' + str(kpiTreat0Post0_mean) + \
+            ', Post-treatment mean KPI of comparison group: ' + str(kpiTreat0Post1_mean) + \
+            '}\n{ Pre-treatment mean KPI of treated group: ' + str(kpiTreat1Post0_mean) + \
+            ', Post-treatment mean KPI of treated group: ' + str(kpiTreat1Post1_mean) + \
+                '}')
 
     def calculatePercentImprovement(self, df):
         '''
         Calculates and prints out the percent improvement of the treated group in comparison to comparison group.
         '''
         treatmentMatrix = self._calculate_aggregate_treatment_matrix(df, aggregate=np.mean)
         kpiTreat0Post0_mean, kpiTreat0Post1_mean, kpiTreat1Post0_mean, kpiTreat1Post1_mean = treatmentMatrix
-        inital = kpiTreat1Post0_mean + (kpiTreat0Post1_mean - kpiTreat0Post0_mean)
+        initial = kpiTreat1Post0_mean + (kpiTreat0Post1_mean - kpiTreat0Post0_mean)
         percentImprovement = ((kpiTreat1Post1_mean - initial) / initial) * 100
         return percentImprovement
 
     def printPercentImprovement(self, df):
         '''
         Prints percent improvent of the treated group in comparison to comparison group.
         '''
-        percentImprovement = calculatePercentImprovement(self, df)
-        print('percent improvement: ' + str(percentImprovement) + '%')
+        percentImprovement = self.calculatePercentImprovement(df)
+        self.printTreatmentMatrixPoints(df)
```

Comment:
please change function name if we want to keep this line

Reply:
I separated the output to be print treatment matrix and print percent improvement